src/functional/iterator/embedded.py

```diff
@@ -701,25 +699,57 @@ def _get_axes(
     )
 
 
+def _single_vertical_idx(indices, column_axis_idx, column_index):
+    transformed = tuple(
+        index if i != column_axis_idx else column_index for i, index in enumerate(indices)
+    )
+    return transformed
+
+
 def _make_tuple(
     field_or_tuple: LocatedField | tuple,  # arbitrary nesting of tuples of LocatedField
```

Comment:
Why not
```suggestion
    field_or_tuple: LocatedField | tuple[LocatedField | Column, ...],  # arbitrary nesting of tuples of LocatedField
```
?

Reply:
because it's could recursively contain tuples

Reply:
Oops, yes, you're right, there was a bug in my suggestion. I meant:
```suggestion
    field_or_tuple: LocatedField | tuple[LocatedField | tuple, ...],  # arbitrary nesting of tuples of LocatedField
```
Of course it doesn't capture the full recursive definition, only one level, but maybe it's enough as documentation hint and for the most simple errors. Anyway, it's not something critical, feel free to ignore.